### ISSUE TITLE
JEPA experiments wt comprssion

### ISSUE THREAD

**Issue by kctezcan** [ecmwf/WeatherGenerator#1867]
### Describe the task. It can be a feature, documentation, etc.

We need to see if JEPA can learn useful representations. To test this we will pre-train models with compression and finetune these with simple decoders to see if the representation is useful. 

1. usual settings, healpix level=5 and latent dim=2048 give an expansion rate of (12288*2048)/(40320*105)=~7.
2. setting healpix level=4 and latent dim=512 yields a compression of (3072*512)/(40320*105)=0.37, i.e. ~x3 compression
3. setting healpix level=4 and latent dim=128 yields a compression of (3072*512)/(40320*105)=0.09, , i.e. ~x11 compression
4. setting healpix level=4 and latent dim=64 yields a compression of (3072*512)/(40320*105)=0.045, , i.e. ~x22 compression

### Area

- [ ] datasets, data readers, data preparation and transfer
- [x] model
- [x] science
- [ ] infrastructure and engineering
- [ ] evaluation, export and visualization
- [ ] documentation

**Comment by clessig** [comment]
We don't have 105 channels in the ERA5 dataset, and use 83 or 84. But roughly speaking, the math is valid.
